README.md

```diff
@@ -1,18 +1,115 @@
-- [installing it in the linux subsystem for windows](https://github.com/JMoVS/installing_video_transcoding_on_windows/blob/master/lxss_method.md)
+### Windows 10
+If you're using an up-to-date version of Windows 10, you have access to "[Bash on Ubuntu on Windows](https://msdn.microsoft.com/en-gb/commandline/wsl/about)". This feature allows you to run command line Linux programs on Windows without using a virtual machine, and it also gives us access to Ubuntu's extensive package archives. Everything we need to run these scripts is easily installable with apt-get.
 
-## Notes about the two methods:
+1. First up, you need to activate Bash.
+    1. Open Settings then go to `Update & Security -> For Developers`
+    1. Change your computer to "Developer Mode"
+    1. Click `Start` then look for `Turn Windows features on or off`
+    1. Select `Windows Subsystem for Linux (beta)`
+    1. Click ok. You'll need to restart your PC
+    1. Now open up `cmd.exe` and run `bash`. Follow the instructions then bash is set up and ready to go 
+1. Next we install the dependencies for `video_transcoding`
+    1. Fire up bash
+    1. Run `sudo apt-get update && sudo apt-get upgrade` to ensure everything's up to date
+    1. Run `sudo apt-get install ruby ffmpeg mp4v2-utils mkvtoolnix`
+    1. Install HandBrakeCLI
+        1. `sudo add-apt-repository ppa:stebbins/handbrake-releases`
+        1. `sudo apt-get update`
+        1. `sudo apt-get install handbrake-cli`
+1. Now we can install `video_transcoding`
+    1. `sudo gem install video_transcoding`
 
-### Manually installing binaries
+The `detect-crop` command has an optional dependency on `mpv` for previewing crops. You can't launch graphical Linux apps using "Bash on Ubuntu on Windows", but it is capable of launching Windows programs; thos means we can install `mpv` for Windows then use an alias to launch it from bash:
 
-This option is recommended for most users. You will get a working system with all the binaries in place that will do the job. Updating the dependencies isn't automated though; you'll need to go back to each project's respective download page again and manually check if there's an update. As the dependencies aren't updated that often, this shouldn't really be a problem.
+1. Create `C:\bin` and [add it to your path](#adding-a-folder-to-your-path)
+1. Download a Windows build of [mpv](https://mpv.io/) from [here](https://mpv.srsfckn.biz/)
+1. Extract it to `C:\bin`
+1. Create an alias for mpv:
+    `echo "alias mpv=mpv.exe" >> .bashrc`
 
-### Installing it in the linux subsystem for windows
+If you have your movie files in the Windows file system (so NOT in `~userfolder/AppData/Local/lxss/home`), for example on your desktop, you have to provide the path to it via `/mnt/[drive letter]/path/to/your/file/location`. You can also navigate to the folder in Explorer, then type bash in the address bar to open a bash instance to that location
```

Comment:
Super pedantic little things: Are there spaces after the `s ? Do you think it looks better with a soace? I trust your judgement there.